index.html

```diff
@@ -384,7 +384,7 @@
           <dfn title="pointerLockElement"></dfn>
 
           <p>While the pointer is locked, returns the result of the
-          <a href="https://w3c.github.io/webcomponents/spec/shadow/#dfn-retargeting-algorithm">retargeting algorithm</a>
+          <a href="https://dom.spec.whatwg.org/#retarget">retargeting algorithm</a>
```

Comment:
Could you use the similar wording as DOM standard uses?

> Let relatedTarget be the result of invoking retargeting event’s relatedTarget against target if event’s relatedTarget is non-null, and null otherwise.

You can use this sentense as a reference.